ssvc-calc/CISA-Coordinator-v2.0.3.json

```diff
@@ -64,17 +64,17 @@
 		{
 		    "label": "Minimal",
 		    "key": "M",
-		    "description": "Type of harm is \"All\" (Physical, Environmental,Financial,Psychological). The effect is below the threshold for all aspects described in material."
+		    "description": "Type of harm is \"All\" (Physical, Environmental,Financial,Psychological) and the associated Safety Impact Value is \"None\". The effect is below the threshold for all aspects described in material."
```

Comment:
I think this should say:
For any type of harm (Physical, Environmental,Financial,Psychological) the Safety Impact value is \"None\" or \"Minor\".
This also means you can remove the trailing "The effect is below..." sentence because the description for material just says the value of safety impact is Major, and None/Minor fully captures what it means for the value to be below major.